e2e/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const PRODUCT_NAME = process.env.PLAYWRIGHT_PRODUCT_NAME;
+export const HOMEPAGE_ENDPOINT = '/';
+export const COLLECTIONS_PAGE_ENDPOINT = `/collections/${process.env.PLAYWRIGHT_COLLECTION_NAME}`;
```

Comment:
`process.env.PLAYWRIGHT_COLLECTION_NAME` needs to be tested for, and if it doesn't exist, the test that depends on it should be skipped. Just like with the product name.

You probably want to export from here `export const COLLECTION_NAME = process.env.PLAYWRIGHT_COLLECTION_NAME;` then whereever you use this, something like 

```
if (!COLLECTION_NAME) {
  test.skip('No PLAYWRIGHT_COLLECTION_NAME defined');
  return;
}

const collectionsPageEndpoint = `/collections/${COLLECTION_NAME}`
```

You could also make a simple `getCollectionEndpoint` fn if you want to avoid the duplicated code and string template.

```
export function getCollectionEndpoint(collectionName: stirng) {
  return `/collections/${collectionName}`
}
```